### ISSUE TITLE
readline support on windows cmd.exe is broken

### ISSUE THREAD

**Issue by evilsocket** [evilsocket/bettercap-ng#60]
On Windows, readline doesn't work very well if bettercap is invoked from the cmd.exe, while it works if used via cygwin, mingw, etc.

**Comment by Droid-MAX** [comment]
Just user cmder.
http://cmder.net/

**Comment by evilsocket** [comment]
Yes, as I already wrote, the tool is known to work correctly on other terminals (cmder, cygwin, mingw and so on), this is specifically an issue with cmd.exe.

**Comment by leonardoporpora** [comment]
could this help? https://iojs.org/download/release/v2.0.1/win-x64/

**Comment by evilsocket** [comment]
@leonardoporpora not following, how a javascript runtime can help to fix a `readline` OS compatibility issue?

**Comment by leonardoporpora** [comment]
https://github.com/nodejs/node/issues/2996 I saw a similar problem on github

**Comment by AnthonyMastrean** [comment]
I'm on Windows 1903 and I don't currently see any problems. Colors, fancy chars, and key sequences (like `CTRL+i` back search) work.

![image](https://user-images.githubusercontent.com/217842/61083685-5bdef280-a3fa-11e9-86e8-d16de239e1bc.png)

The Windows console team shipped [_major_ changes](https://devblogs.microsoft.com/commandline/windows-command-line-unicode-and-utf-8-output-text-buffer/) in both 1809 and 1903.

**Comment by evilsocket** [comment]
you are using docker, it should be different than just running it via cmd.exe, which is what this issue is referring to

**Comment by AnthonyMastrean** [comment]
Oh, that's wild. I didn't think that a Docker command could replace like the underlying console's capabilities... I know that if I run a _different_ terminal like mingw, things are _different_. I can't really get bettercap running properly directly on Windows yet, but it's not broken like in #315.

![image](https://user-images.githubusercontent.com/217842/61132668-d9057880-a489-11e9-803e-4933e4642960.png)

It even warns about colors, so it seems like bettercap is properly detecting the console's capabilities?

> WARNING: This terminal does not support colors, view will be very limited.
